src/main/modules/app-insights/index.ts

```diff
@@ -6,13 +6,37 @@ declare class AppInsightsConfiguration {
   roleName: string
 }
 
+function hideUuidInUrlIfNotStaticFile (url: string): string {
+  let fileRegexp = new RegExp('(\\..{2,5}$)')
+  let uuidRegexp = new RegExp('[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89aAbB][a-f0-9]{3}-[a-f0-9]{12}')
+
+  if (url.match(fileRegexp) == null) {
```

Comment:
I think .test is more appropriate here:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test